src/main/java/frc/robot/Constants.java

```diff
@@ -50,12 +50,30 @@ public final class Constants {
     public static final int operatorId = 1;
 
     public static final boolean tuningMode = false;
+    /**
+     * Current threshold that indicates an algae is in possestion
+     */
+    public static final double HAS_ALGAE_CURRENT_THRESHOLD = 0;
 
-
+    /**
+     * Algae misc values
+     */
+    public static final class Algae {
+        public static final int VOLTAGE = 0;
+        public static final int NEGATIVE_VOLTAGE = 0;
+        public static final int BEAM_BRAKE_ID = 0;
+    }
     /**
      * Motor CAN id's.
      */
+
```

Comment:
thats not what it meant by classes separated by a space, it meant line 66 should have a space above it, not between the javadoc and the item its documenting